view/frontend/web/internals/common.js

```diff
@@ -566,9 +566,9 @@ requirejs(['algoliaBundle'], function(algoliaBundle) {
 					const portWithPrefix = port === '' ? '' : `:${port}`;
```

Comment:
Could you fix this "backticked" string as well? :)

Reply:
You got it!